05-how-to-contribute-to-open-source.md

```diff
@@ -133,6 +133,30 @@ Replace "your-username" with your GitHub username.
 
 Congratulations! You just made an open source contribution! In the next chapter, we'll add this to an open source portfolio: a resume for open source contributions through which you can showcase your work and attract potential employers and friends using OpenSauced. For now, let's wrap up with some notes on onboarding.
 
+## Understanding Merge Conflicts
+
+- Merge conflicts happen when two branches have made different changes to the same lines in a file. Git cannot automatically determine which change to keep, and this results in a conflict.
+- When a merge conflict occurs, Git adds special markers (<<<<<<<, =======, and >>>>>>) to the conflicted part of the file, indicating the conflicting sections from different branches.
+
+## Resolving Merge Conflicts in VS Code
+
+### Prerequisites
+
+Ensure you have Git and the GitLens extension installed in VS Code. GitLens is a powerful Git extension that enhances Git-related features in VS Code.
```

Comment:
Do we want to mention GitLens in this section?

Because if I read thoroughly, the instruction here is to resolve the conflicts manually and nothing mention about how to use GitLens.

Reply:
I think we can remove GitLens @adiati98. They can learn about that on their own. We don't want to spoon feed everything.